docs/dev/new_check_howto.md

```diff
@@ -300,9 +300,59 @@ The `logos` folder must contain **three** images with filenames and sizes that m
 
 ### Metadata
 
-Review the contents of `manifest.json` and `metadata.csv`. The metadata catalog is not currently automatically generated, so filling it out manually is a crucial part of the release process. Our example check doesn't send any metrics however, so in this case we will leave it empty.
-
-Our check sends a Service Check though, so we need to add it to the `service_checks.json` file:
+Review the contents of `manifest.json` and `metadata.csv`. The metadata catalog is not currently automatically generated, so filling it out manually is a crucial part of the release process. 
+
+
+#### manifest.json
+
+Find below the complete list of mandatory and optional attributes for your `manifest.json` file:
+
+| Attribute           | Type            | Mandatory/Optional | Description                                                                                                                                                                                                                        |
+| ------              | ----            | ------             | ---------                                                                                                                                                                                                                          |
+| `categories`        | Array of String | Mandatory          | Integration categories used on the [Public documentation Integrations page][12].                                                                                                                                                   |
+| `creates_events`    | Boolean         | Mandatory          | If the integration should be able to create events. If this is set to `false`, attempting to create an event from the integration results in an error.                                                                             |
+| `display_name`      | String          | Mandatory          | Title displayed on the corresponding integration tile in the Datadog application and on the[Public documentation Integrations page][12]                                                                                            |
+| `doc_link`          | String          | Mandatory          | Link to the documentation page of the integration. Format is `https://docs.datadoghq.com/integrations/<INTEGRATION_NAME>`                                                                                                          |
```

Comment:
Some of them like this one are not relevant in the case of `integration-extras`, what is the alternative there. Where do we document the difference?

Reply:
for now there is now difference in term of usage between core and extras

Reply:
edit: `doc_link` logic should be removed from the doc website soon.